src/code42cli/cmds/alerts.py

```diff
@@ -155,15 +155,22 @@ def clear_checkpoint(state, checkpoint_name):
 @alerts.command()
 @alert_options
 @search_options
+@click.option(
+    "--or-query", is_flag=True, cls=searchopt.AdvancedQueryAndSavedSearchIncompatible
+)
 @opt.sdk_options
-def search(cli_state, format, begin, end, advanced_query, use_checkpoint, **kwargs):
+def search(
+    cli_state, format, begin, end, advanced_query, use_checkpoint, or_query, **kwargs
+):
     """Search for alerts."""
     output_logger = logger_factory.get_logger_for_stdout(format)
     cursor = _get_alert_cursor_store(cli_state.profile.name) if use_checkpoint else None
     handlers = ext.create_handlers(
         cli_state.sdk, AlertExtractor, output_logger, cursor, use_checkpoint
     )
     extractor = _get_alert_extractor(cli_state.sdk, handlers)
+    if or_query:
```

Comment:
any reason not to just do `extractor.use_or_query = or_query` ?

Reply:
nope, I think i just copied the logic from security-data and then removed the extra line.